.github/ISSUE_TEMPLATE/NEW_RULE.md

```diff
@@ -8,18 +8,20 @@ assignees: ''
 ---
 
 <!--
-    ESLint adheres to the [JS Foundation Code of Conduct](https://js.foundation/community/code-of-conduct).
+    ESLint adheres to the Open JS Foundation Code of Conduct:
+    https://eslint.org/conduct
 
-    This template is for new rule proposals. If you are proposing a new rule, please continue on. If you are here for another reason, please see below:
+    This template is for bug reports. If you are here for another reason, please see below:
```

Comment:
Is this change accurate? It looks like all the templates now say they are intended for bug reports.

Reply:
Oops, copy-paste error! Nice catch.